tests/iso-8859-2-hungarian/auto-apro.hu.xml

```diff
@@ -1,474 +0,0 @@
-<?xml version="1.0" encoding="iso-8859-2" ?>
```

Comment:
Please don't convert the XML test files to plaintext.  Part of chardet attempts to remove tags automatically, so these are important test cases.